deepchem/models/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,95 @@
+try:
+  import torch
+  import pytorch_lightning as pl  # noqa
+  PYTORCH_LIGHTNING_IMPORT_FAILED = False
+except ImportError:
+  PYTORCH_LIGHTNING_IMPORT_FAILED = True
+
+
+class DCLightningModule(pl.LightningModule):
+  """DeepChem Lightning Module to be used with Lightning trainer.
+
+  Example code
```

Comment:
The formatting here and elsewhere in the docstring has issues. Are you planning to fix in #2958 @Chahalprincy? This new class also needs to be added to the docs/ folder to render on readthedocs

Reply:
Yes, I will do it in another PR. Sorry, I missed it.